lib/libconfig.h

```diff
@@ -57,6 +57,7 @@ extern "C" {
 
 #define CONFIG_FORMAT_DEFAULT  0
 #define CONFIG_FORMAT_HEX      1
+#define CONFIG_FORMAT_BIN      2
```

Comment:
the C++ wrapper needs to be updated as well to update the corresponding enum there